app/modules/notifications/models.py

```diff
@@ -85,6 +108,16 @@ def __init__(self, sender, multiple=False):
     def set_collaboration(self, collab):
         self.data['collaboration_guid'] = collab.guid
 
+    def set_individual_merge_request(self, individuals, encounters, request_data):
```

Comment:
likewise if this is used for the request and the complete notification, the word request here is misleading.